### DISCUSSION TITLE
Regarding CSS class `code-frame frame`

### DISCUSSION THREAD

**Discussion by serhii-bilyk-intellias** [mdx-js/mdx#2348] [Q&A]
Could you please explain how did you made `code-frame`
Please see [this page](https://mdxjs.com/docs/using-mdx/)
I am interested in this code block
![image](https://github.com/mdx-js/mdx/assets/144336942/62c9a0b5-a899-45aa-839a-f1f53a3ac52a)

I see that you are [using](https://github.com/mdx-js/mdx/blob/main/docs/docs/using-mdx.server.mdx) mdx attribute 
```
```mdx path="example.mdx"
export const Thing = () => <>World</>

# Hello <Thing />
```
```
How can I make this piece of code reusable? 

I was trying to copy and paste it but of course it does not work. I need to create a code block where user can choose language with copy button

**Comment by wooorm** [comment]
> How can I make this piece of code reusable? 

I don‘t understand what you are asking

---

In think you are looking for:
https://github.com/mdx-js/mdx/blob/07e2b36edf6bbbf7b1c94134a8ab06a34967010d/website/mdx-config.js#L192

But you can also read the docs: https://mdxjs.com/guides/syntax-highlighting/#syntax-highlighting-with-the-meta-field
